### PR TITLE
Add docs related giving data source support to mi dashboard

### PR DESCRIPTION
**Pull request by malakaganga**
## Purpose
Related To: https://github.com/wso2/api-manager/issues/291

### CODE REVIEW

**Comment by SanojPunchihewa** [comment]
Shall we add a note about the U2 level this feature is available?

File: en/docs/install-and-setup/install/installing-the-product/setting-up-ibm-db2.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
+# Setting up an IBM DB2
+
+Follow the steps given below to set up the required IBM databases for your Micro Integrator (MI) Dashboard.
+
+## Prerequisites
+
+Install DB2. Following steps are tested in DB2 v11.5.7.0
```
Comment (samgnaniah) [inline en/docs/install-and-setup/install/installing-the-product/setting-up-ibm-db2.md:7]:
```suggestion
Install DB2. The following steps are tested in DB2 v11.5.7.0.
```

File: en/docs/install-and-setup/install/installing-the-product/setting-up-ibm-db2.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+username = "db2inst1"
+password = "xxxx"
+driverClassName = "com.ibm.db2.jcc.DB2Driver"
+maximumPoolSize = "100"
+poolName = "dashboard-1"
+connectionTimeout = "400000"
+maxLifetime = "2000000"
+```
+
+See the descriptions of [database connection parameters]({{base_path}}/reference/config-catalog-mi-dashboard/#database-connection).
```
Comment (samgnaniah) [inline en/docs/install-and-setup/install/installing-the-product/setting-up-ibm-db2.md:45]:
```suggestion
For more information, see the descriptions of [database connection parameters]({{base_path}}/reference/config-catalog-mi-dashboard/#database-connection).
```

File: en/docs/install-and-setup/install/installing-the-product/setting-up-mssql.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+GO
+CREATE DATABASE dashboarddb;
+GO
+
+Change the <DB_NAME> in first line of mssql.sql script given in <MI_DASHBOARD_HOME>/dbscripts/mssql/mssql.sql to 
+actual DB name 
+ex: use dashboarddb;
+
+:r <MI_DASHBOARD_HOME>/dbscripts/mssql/mssql.sql
+```
```
Comment (samgnaniah) [inline en/docs/install-and-setup/install/installing-the-product/setting-up-mssql.md:28]: Shall we break this down into multiple steps and code blocks?

File: en/docs/install-and-setup/install/installing-the-product/setting-up-mysql.md [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+username = "root"
+password = "root"
+driverClassName = "com.mysql.cj.jdbc.Driver"
+maximumPoolSize = "100"
+poolName = "dashboard-1"
+connectionTimeout = "400000"
+maxLifetime = "2000000"
+```
+
+See the descriptions of [database connection parameters]({{base_path}}/reference/config-catalog-mi-dashboard/#database-connection).
```
Comment (samgnaniah) [inline en/docs/install-and-setup/install/installing-the-product/setting-up-mysql.md:52]:
```suggestion
For more information, see the descriptions of [database connection parameters]({{base_path}}/reference/config-catalog-mi-dashboard/#database-connection).
```

File: en/docs/install-and-setup/install/installing-the-product/setting-up-oracle.md [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+username = "SYSTEM"
+password = "oracle"
+driverClassName = "oracle.jdbc.OracleDriver"
+maximumPoolSize = "100"
+poolName = "dashboard-1"
+connectionTimeout = "400000"
+maxLifetime = "2000000"
+```
+
+See the descriptions of [database connection parameters]({{base_path}}/reference/config-catalog-mi-dashboard/#database-connection).
```
Comment (samgnaniah) [inline en/docs/install-and-setup/install/installing-the-product/setting-up-oracle.md:41]:
```suggestion
For more information, see the descriptions of [database connection parameters]({{base_path}}/reference/config-catalog-mi-dashboard/#database-connection).
```

File: en/mkdocs.yml [hunk 6]
```diff
@@ -2160,6 +2166,5 @@ extra:
         - type: linkedin
           link: https://www.linkedin.com/company/wso2
     site_version: 4.0.0
-    #base_path: http://localhost:8000
```
Comment (samgnaniah) [inline en/mkdocs.yml:2163]: Shall we revert this? It's commented out for ease of use when checking locally.

File: en/docs/install-and-setup/install/installing-the-product/setting-up-postgresql.md [hunk 7]
```diff
@@ -0,0 +1,49 @@
...
+username = "postgres"
+password = "root"
+driverClassName = "org.postgresql.Driver"
+maximumPoolSize = "100"
+poolName = "dashboard-1"
+connectionTimeout = "400000"
+maxLifetime = "2000000"
+```
+
+See the descriptions of [database connection parameters]({{base_path}}/reference/config-catalog-mi-dashboard/#database-connection).
```
Comment (samgnaniah) [inline en/docs/install-and-setup/install/installing-the-product/setting-up-postgresql.md:49]:
```suggestion
For more information, see the descriptions of [database connection parameters]({{base_path}}/reference/config-catalog-mi-dashboard/#database-connection).
```

**Comment by malakaganga** [comment]
> Shall we add a note about the U2 level this feature is available?

Added

File: en/mkdocs.yml (hunk 6 above)
Comment (malakaganga) [reply en/mkdocs.yml:2163]: Fixed

**Review by samgnaniah — APPROVED** [review]

**Pull request merged**